apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -593,6 +578,11 @@ rpc_call(Node, Fun, Args) ->
         Res -> Res
     end.
```

Comment:
Suggestion: maybe we could remove this `rpc_call/3` function.

Reply:
Yes, that's the intention, but the rest of the work here requires more thorough refactoring.